frontend/src/metabase/actions/containers/ActionParametersInputForm/ActionParametersInputForm.unit.spec.tsx

```diff
@@ -139,6 +139,18 @@ describe("Actions > ActionParametersInputForm", () => {
 
     const idParameter = createMockActionParameter({ id: "id" });
 
+    const parameter1 = createMockActionParameter({
+      id: "parameter_1",
+      type: "type/Text",
+      "display-name": "Parameter 1",
+    });
+
+    const parameter2 = createMockActionParameter({
+      id: "parameter_2",
+      type: "type/Text",
+      "display-name": "Parameter 2",
+    });
```

Comment:
Action parameters for implicit actions now include "display-name". Note this isn't the case for query actions, so I had to create separate parameters to the original ones here:
https://github.com/metabase/metabase/blob/ada9bbc1672ac3a48200d37821ec0666a3d41bd7/frontend/src/metabase/actions/containers/ActionParametersInputForm/ActionParametersInputForm.unit.spec.tsx#L24-L32